test/lender/Lender.sol

```diff
@@ -149,8 +166,11 @@ contract Lender {
     require(token.underlying() == u, '');
     require(token.unlockTimestamp() == m, '');
 
+    // Transfer fee to illuminate
+    Safe.transferFrom(IErc20(u), msg.sender, illuminate, a/feenominator);
+
     // Transfer underlying token from user to illuminate
-    Safe.transferFrom(IErc20(u), msg.sender, address(this), a);
+    Safe.transferFrom(IErc20(u), msg.sender, address(this), a - a / feenominator);
```

Comment:
order of ops worries me here, we usually strictly wrap stuff in parens. i don't trust .sol